linux/net/rina/rds/ringq.c

```diff
@@ -165,7 +165,7 @@ int ringq_destroy(struct ringq * q,
 
         q_destroy(q, dtor);
 
-        LOG_DBG("RING Queue %pK destroyed successfully", f);
+        LOG_DBG("RING Queue %pK destroyed successfully", q);
```

Comment:
AFAIK it is a convention to print Kernel pointers:
https://www.kernel.org/doc/Documentation/printk-formats.txt

Francesco introduced it in the origin of times :)

2016-08-22 16:56 GMT+02:00 Vincenzo Maffione <notifications@github.com>:

> In linux/net/rina/rds/ringq.c
> <https://github.com/IRATI/stack/pull/986#discussion_r75693021>:
>
> > @@ -165,7 +165,7 @@ int ringq_destroy(struct ringq * q,
> >
> >          q_destroy(q, dtor);
> >
> > -        LOG_DBG("RING Queue %pK destroyed successfully", f);
> > +        LOG_DBG("RING Queue %pK destroyed successfully", q);
>
> Why the "K" character if q is a pointer?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/IRATI/stack/pull/986/files/190c526ec03f7d58cd13476b9bd0ce67c12c08e9#r75693021>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABtu_DDwA3XMrlNY8Ox57aWfW9s2H1KIks5qibitgaJpZM4JnRhy>
> .
>



-- 
Leonardo Bergesio
Distributed Applications and Networks Area - DANA
i2CAT Foundation - www.i2cat.net
C/ Gran Capità 2-4, office 109, Nexus I building
08034 Barcelona, Catalonia (Spain)
T: +34 93 567 9926
F: +34 93 553 2520
W: http://dana.i2cat.net